package.json

```diff
@@ -22,6 +22,7 @@
     "build-ga2-db": "esrun catalog/ga2/build/ts/build-catalog.ts",
     "build-brc-from-ncbi": "python3 -m catalog.build.py.build_files_from_ncbi",
     "build-ga2-from-ncbi": "python3 -m catalog.ga2.build.py.build_files_from_ncbi",
+    "merge-taxonomy-read-counts": "python3 -c \"import json, sys; f1=json.load(open('catalog/build/intermediate/taxIdReadCount.json')); f2=json.load(open('catalog/ga2/build/intermediate/taxIdReadCount.json')); dups=set(f1.keys())&set(f2.keys()); (print(f'Warning: Duplicate taxonomy IDs found: {sorted(dups)} (ga2 values will be used)', file=sys.stderr) if dups else None); json.dump({**f1,**f2}, open('app/components/Entity/components/ConfigureWorkflowInputs/components/Main/components/Stepper/components/Step/SequencingStep/components/ENASequencingData/hooks/UseENADataByTaxonomyId/taxonomy_read_counts.json','w'), indent=2)\"",
```

Comment:
I'd maybe like to see this bit of python live somewhere else and be a bit cleverer. BRC and ga2 should have the same counts for a given taxid right? So it could validate that before tossing one out for ex. Or if they don't match bc one was run more recently than the other, take the larger value maybe. I'd assume that's both the more accurate and more troublesome case.